reservation.yaml.sample

```diff
@@ -1,9 +1,13 @@
 ---
 # ############################################### #
-# Grid'5000 reservation parameters                #
+# Enos Configuration File                         #
 # ############################################### #
 
 name: OpenStack/Discovery-Enos2
+
+# Resources Provider. Values are: g5k, vbox
```

Comment:
We should document somewhere the other form:
```
provider:
  type:
```
with the fact that custom keys can be provided and are provider-specific.